lib/user_repository.rb

```diff
@@ -1,9 +1,9 @@
-require 'user'
-require 'database_connection'
+require "user"
```

Comment:
Me and Ev think this needs to be require_relative to work in rackup

Reply:
forgot to update this file, my bad

Reply:
Yes I think so. `require_relative './user'`

Reply:
it's not best practice btw, i just have a formatter which changes all single quotes to double